sys/dev/ath/ath_hal/ar9002/ar9271_attach.c

```diff
@@ -442,13 +442,13 @@ ar9271FillCapabilityInfo(struct ath_hal *ah)
 
 	if (!ar5416FillCapabilityInfo(ah))
 		return AH_FALSE;
-	pCap->halNumGpioPins = 12;
-	pCap->halWowSupport = AH_TRUE;
-	pCap->halWowMatchPatternExact = AH_TRUE;
-#if 0
-	pCap->halWowMatchPatternDword = AH_TRUE;
-#endif
-	/* AR9285 has 2 antennas but is a 1x1 stream device */
+	pCap->halNumGpioPins = (AH_PRIVATE(ah)->ah_isusb) ? 16 : 12;
```

Comment:
Because of: `sc->ngpiopins = (sc->flags & ATHN_FLAG_USB) ? 16 : 12;` in athn code